tests/Kron.py

```diff
@@ -259,7 +356,7 @@ def measureKronInPython(self, objImg, xcen, ycen, nsigma, kfac, nIterForRadius,
 
         return R_K, sumI, 0, False
 
-    def testEllipticalGaussian(self):
+    def xtestEllipticalGaussian(self):
```

Comment:
I assume the 'x' is a cheap & cheerful way of skipping the test, but you'll need to explain properly why this test code should be included by not executed.